app/i18n/en/gen.php

```diff
@@ -156,6 +156,7 @@
 		'damn' => 'Damn!',
 		'default_category' => 'Uncategorized',
 		'no' => 'No',
+		'not_applicable' => 'N/A',
```

Comment:
I'm not fan of "N/A", I would prefer complete words such as "Not available" or maybe the more explicit "URL cannot be configured" (but it's a bit too long)